infrastructure/swagger/05_paths.yaml

```diff
@@ -343,6 +343,49 @@ paths:
         - ${authoriser_name}: []
         - app-level0: []
 
+  ? /ProductTeam/{product_team_id}/Product/{product_id}/DeviceReferenceData/MhsMessageSet
+  : post:
+      operationId: createDeviceReferenceDataMessageSet
+      summary: createDeviceReferenceDataMessageSet endpoint for APIGEE integration
+      parameters:
+        - name: product_team_id
+          in: path
+          required: true
+          description: logical identifier
+          schema:
+            type: string
+        - name: product_id
+          in: path
+          required: true
+          description: logical identifier
+          schema:
+            type: string
+        - *RequestHeaderVersion
+        - *RequestHeaderRequestId
+        - *RequestHeaderCorrelationId
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                questionnaire_responses:
+                  type: object
+                  description: Questionnaire Responses for MHS Message Set questionnaire
+      responses:
+        responses:
```

Comment:
Need to create a response like in the swagger ticket.

Reply:
i didn't want to conflict with the swagger ticket by making changes here, depending on what goes in first i'd say best to put in as a fix-forward?

Reply:
Yeah, not sure what the best option is here. I guess you are right. I'll fix it in my swagger ticket